versionist.conf.js

```diff
@@ -1,32 +1,59 @@
-module.exports = {
+var _ = require('lodash');
+var execSync = require('child_process').execSync;
 
-  subjectParser: 'angular',
+var getAuthor = (commitHash) => {
+  return execSync(`git show --quiet --format="%an" ${commitHash}`, { encoding: 'utf8' }).replace('\n', '');
+}
 
+module.exports = {
+  // This setup allows the editing and parsing of footer tags to get version and type information,
+  // as well as ensuring tags of the type 'v<major>.<minor>.<patch>' are used.
+  // It increments in a semver compatible fashion and allows the updating of NPM package info.
+  editChangelog: true,
+  parseFooterTags: true,
   getGitReferenceFromVersion: 'v-prefix',
+  incrementVersion: 'semver',
+  updateVersion: 'npm',
 
+  // Always add the entry to the top of the Changelog, below the header.
   addEntryToChangelog: {
     preset: 'prepend',
-    fromLine: 5
+    fromLine: 6
   },
 
-  updateVersion: 'npm',
-
+  // Only include a commit when there is a footer tag of 'change-type'
+  // or 'changelog-entry'.
+  // Ensures commits which do not up versions are not included.
   includeCommitWhen: (commit) => {
-    return commit.footer['changelog-entry'];
+    return !!commit.footer['change-type'] || !!commit.footer['changelog-entry'];
```

Comment:
Yep, I've mixed my notes up. :-/ For some reason the ones that ended up on the TODO list included single changelog entry commits, but having gone back, the call notes do indeed specify a commit must have both (if there's a `changelog-entry` at all).
Sorry, there's been so much changed over the last two weeks it's been tricky keeping everything aligned.